sw/host/spiflash/verilator_spi_interface.h

```diff
@@ -12,26 +12,27 @@
 namespace opentitan {
 namespace spiflash {
 
-// Implements SPI interface for an OpenTitan instance running on Verilator.
-// The OpenTitan Verilator model provides a file handle for the SPI device
-// interface. This class sends ands recevies data to the device handle, and
-// adds synchronication delays between writes and reads.
-// This class is not thread safe.
+/**
+ * Implements SPI interface for an OpenTitan instance running on Verilator.
+ * The OpenTitan Verilator model provides a file handle for the SPI device
+ * interface. This class sends ands recevies data to the device handle, and
+ * adds synchronication delays between writes and reads.
+ * This class is not thread safe.
+ */
 class VerilatorSpiInterface : public SpiInterface {
  public:
-  // Constructs instance pointing to the `spi_filename` file path.
+  /** Constructs instance pointing to the `spi_filename` file path. */
   explicit VerilatorSpiInterface(std::string spi_filename)
       : spi_filename_(spi_filename), fd_(-1) {}
 
-  // Closes the internal file handle used to communicate with the SPI device.
+  /**
+   * Closes the internal file handle used to communicate with the SPI device.
+   */
   ~VerilatorSpiInterface() override;
 
-  // Initialize interface.
+  // SpiInterface functions.
```

Comment:
Drop this comment. It merely says what, but not why.

Reply:
Done.